spec/phase2.html

```diff
@@ -297,6 +320,766 @@
           <li><code>"scroll"</code>
           <li><code>"zoom"</code>
         </ul>
+    </section>
+      <section>
+        <h3>Platform compatibility table</h3>
+
+        <p>
+          The following table shows proposed accessible actions and
+          the native actions or methods they correspond to on each platform
+          with an accessibility API.
+        </p>
+        <p>
+          When an action maps cleanly to a method or action on all or nearly
+          all platforms, that's good evidence that it's a good candidate for
+          an accessible action in the Accessibility Object Model.
+          When an action doesn't map cleanly, that means it's potentially
+          more risky for compatibility.
+        </p>
+        <p>
+          In the table below, Tier 1 actions appear to have broad support
+          across almost all platforms. Tier 2 actions are supported on
+          at least 2 unrelated platforms. The remaining native actions
+          are shown below.
+        </p>
+        <table>
+        <thead>
+        <tr>
+        <th>Tier</th>
+        <th>Action</th>
+        <th>Mac</th>
+        <th>iOS</th>
+        <th>MSAA + IAccessible2</th>
+        <th>UIA</th>
+        <th>Android</th>
+        <th>ATK</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+        <td>1</td>
+        <td>Activate</td>
+        <td>AXPressAction</td>
+        <td>accessibilityActivate</td>
+        <td>IAccessible::accDoDefaultAction</td>
+        <td><div>InvokeProvider::Invoke<br>IToggleProvider::Toggle</div>
+        </td>
+        <td>ACTION_CLICK</td>
+        <td>atk_action_do_action</td>
+        </tr>
+        <tr>
+        <td>1</td>
+        <td>Focus</td>
+        <td>set AXFocused true</td>
+        <td>
+        </td>
+        <td>IAccessible::accSelect SELFLAG_TAKEFOCUS</td>
+        <td>
+        </td>
+        <td>
+        <div>ACTION_FOCUS<br>ACTION_CLEAR_FOCUS</div>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>1</td>
+        <td>ScrollByPage</td>
+        <td>
+        </td>
+        <td>accessibilityScroll</td>
+        <td>
+        </td>
+        <td>IScrollProvider::Scroll</td>
+        <td>
+        <div>ACTION_SCROLL_BACKWARD<br>ACTION_SCROLL_DOWN<br>ACTION_SCROLL_FORWARD<br>ACTION_SCROLL_LEFT<br>ACTION_SCROLL_RIGHT<br>ACTION_SCROLL_UP</div>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>1</td>
+        <td>ScrollIntoView</td>
+        <td>
+        <div>AXScrollToVisible,<br>set AXVisibleCharacterRange</div>
+        </td>
+        <td>
+        </td>
+        <td>IAccessible2::scrollTo<br>IAccessibleText::scrollSubstringTo</td>
+        <td>IScrollItemProvider::ScrollIntoView</td>
+        <td>ACTION_SHOW_ON_SCREEN</td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>1</td>
+        <td>SecondaryClick</td>
```

Comment:
should you use IAccessibleAction and AtkAction as a fallback whenever the API doesn't provide a better match?